src/runtime/virtcontainers/container.go

```diff
@@ -31,6 +31,8 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+const sensitiveMountOptions string = "sensitiveMountOptions"
```

Comment:
nit: Let's define this in `src/runtime/pkg/direct-volume/utils.go` like for the other similar constants?